packages/block-library/src/classic/edit.js

```diff
@@ -50,12 +50,20 @@ export default class ClassicEdit extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
-		const { clientId, attributes: { content } } = this.props;
-
+		const { clientId, attributes } = this.props;
 		const editor = window.tinymce.get( `editor-${ clientId }` );
+		let { contentRaw } = attributes;
```

Comment:
contentRaw is not persisted, we should probably use "local state" for it.

Reply:
Ah, sure, that'd work even better :)